packages/api/src/middleware/kubernetes.test.ts

```diff
@@ -48,19 +48,22 @@ describe("kubernetes middleware", () => {
     });
 
     it("should return orchestrators from getOrchestrators()", async () => {
-      const response = await req.getOrchestrators();
+      const response = await req.orchestratorsGetters[0]();
```

Comment:
Same, could we keep using `getOrchestrators`? I think there are some other occurrences too.